CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ endif()
 add_executable(gemma run.cc)
 target_sources(gemma PRIVATE ${SOURCES})
 set_property(TARGET gemma PROPERTY CXX_STANDARD 17)
-target_link_libraries(gemma hwy hwy_contrib sentencepiece)
+target_link_libraries(gemma hwy hwy_contrib sentencepiece stdc++fs)
```

Comment:
This is a toolchain requirement only for some old compilers. Either update the compiler, or pass in the link flag from the command line/a toolchain file/a preset via [CMAKE_EXE_LINKER_FLAGS](https://cmake.org/cmake/help/latest/variable/CMAKE_EXE_LINKER_FLAGS.html).